lib/datadog/tracing/tracer.rb

```diff
@@ -362,7 +362,12 @@ def bind_trace_events!(trace_op)
           event_span_op.service ||= @default_service
         end
 
+        events.propagate.subscribe do |_event_span, event_trace_op|
```

Comment:
Curious; why does this even provide an `event_span` parameter? What would that mean? The span that was active when the propagation occurred?

Is this an artifact of a promotion of an `event_span` parameter to an event base class? I'd like to affirm this design before we commit to a particular API for propagation.